structs.go

```diff
@@ -1102,3 +1103,38 @@ const (
 
 	ErrCodeReactionBlocked = 90001
 )
+
+const (
+	IntentsGuilds = 1 << iota
+	IntentsGuildMembers
+	IntentsGuildBans
+	IntentsGuildEmojis
+	IntentsGuildIntegrations
+	IntentsGuildWebhooks
+	IntentsGuildInvites
+	IntentsGuildVoiceStates
+	IntentsGuildPresences
+	IntentsGuildMessages
+	IntentsGuildMessageReactions
+	IntentsGuildMessageTyping
+	IntentsDirectMessagesMessages
```

Comment:
This seems like an unintended stutter; the Discord docs list it as simply `DIRECT_MESSAGES`.

Reply:
Fixed